mozregression/releases.py

```diff
@@ -58,8 +58,10 @@ def date_of_release(release):
     """
     Provide the date of a release.
     """
+    if isinstance(release, str):
+        release = ''.join([i for i in release if i.isdigit()])
```

Comment:
Hm, I don't like this test - if we have a revision, say "aeefd3ae4", it will detect that it is the release 34 :(

Just remove that code, and do an "except KeyError, ValueError" or write a specific try to test the int conversion.